templates/includes/records/hierarchy-global.html

```diff
@@ -1,29 +1,236 @@
+                                <p class="hierarchy-full-panel__text-level">{% if is_current_item %}You are currently looking at{% else %}Within{% endif %} the division: <a href="{% record_url item %}">{{item.reference_number}}</a></p>
+                                <p class="hierarchy-full-panel__text-title">{{item.title}}</p>
+                            </div>
+                        </div>
+                    -->
+
+                    <!-- Level 4 -->
+                    <!-- forloop.counter should be equal to 3 however due to missing data it will be 2 -->
+                    {% elif item.level_code == 3 or forloop.counter == 2 %}
+                        <div class="hierarchy-full-panel__bar-container-outer">
+                            <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-4">
+                                <p><span class="hierarchy-full-panel__bar-number">N</span>
+                                <span class="hierarchy-full-panel__bar-label">records</span></p>
+                            </div>
+                            <div class="hierarchy-full-panel__text-container">
+                                <p class="hierarchy-full-panel__text-level">{% if is_current_item %}You are currently looking at{% else %}Within{% endif %} the series: <a href="{% record_url item %}">{{item.reference_number}}</a></p>
+                                <p class="hierarchy-full-panel__text-title">{{item.title}}</p>
+                            </div>
+                        </div>
+
+                    <!-- Level 5 -->
+                    <!--Commenting out due to missing data
+                    {#{% elif item.level_code == 4 or forloop.counter == 4 %}#}
+                        <div class="hierarchy-full-panel__bar-container-outer">
+                            <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-5">
+                                <p><span class="hierarchy-full-panel__bar-number">N</span>
+                                <span class="hierarchy-full-panel__bar-label">records</span></p>
+                            </div>
+                            <div class="hierarchy-full-panel__text-container">
+                                <p class="hierarchy-full-panel__text-level">{% if is_current_item %}You are currently looking at{% else %}Within{% endif %} the sub-series: <a href="{% record_url item %}">{{item.reference_number}}</a></p>
+                                <p class="hierarchy-full-panel__text-title">{{item.title}}</p>
+                            </div>
+                        </div>
+                    -->
+
+                    <!-- Level 6 -->
+                    <!--As above, should be 5 but missing data prevents this-->
```

Comment:
Is this commented code required, if so could we insert a TODO here for these levels to keep track.